package.json

```diff
@@ -48,5 +48,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "dependencies": {
+    "classnames": "^2.3.1"
```

Comment:
@donghwipark how did this dependency get added?

Reply:
It was occuring error on the local environment. And solved after I include this